package/Dockerfile

```diff
@@ -4,7 +4,9 @@ ARG TARGETPLATFORM
 ARG KUBERNETES_RELEASE=v1.21.3
 WORKDIR /bin
 RUN set -x \
- && curl -fsSLO https://dl.k8s.io/release/${KUBERNETES_RELEASE}/bin/${TARGETPLATFORM}/kubectl \
```

Comment:
@pjbgf - Wondering if doing the check this way is similar enough to what we do in shell here: https://github.com/rancher/shell/blob/2464adbcf9ba4feecd3c8e31ac787de206578ee5/hack/make/deps.mk#L4

That this provides the same level of validation as the shell one? Or in other words, are these essentially equivalent checks?

Reply:
Yep, LGTM.